app/common/delay.ts

```diff
@@ -1,7 +1,24 @@
+import {MaybePromise} from 'app/plugin/gutil';
+
 /**
  * Returns a promise that resolves in the given number of milliseconds.
  * (A replica of bluebird.delay using native promises.)
  */
 export function delay(msec: number): Promise<void> {
   return new Promise<void>((resolve) => setTimeout(resolve, msec));
 }
+
+export async function waitToPass(fn: () => MaybePromise<any>, maxWaitMs: number = 2000) {
```

Comment:
Hmm. A bit unusual to put a test-only method in app? But given that there's nowhere very obvious in tests to put a common library, I guess that's ok. Can you add some documentation about what the method is for (I know there's already undocumented copies of the function lying around, but we should have a higher standard for app/common).

Reply:
You're right. Haven't though about it. I moved it to `test/testUtils`.